src/libs/ReactNativeLauncher.js

```diff
@@ -244,6 +255,116 @@ class ReactNativeLauncher extends Launcher {
     }
   }
 
+  /**
+   * This method returns all cookies when asked by the running webview(s).
+   * This method is callable by the content script.
+   *
+   * @param {String} options.cookieDomain : Domain's name where to get cookies from.
+   * @param {String} options.konnectorSlug : Slug of the konnector triggering the function.
+   * @param {String} options.idAccount : Id of the account triggering the function.
+   */
+  async getCookies(cookieDomain) {
+    const { manifest } = this.startContext
+    if (!manifest.cookie_domains.includes(cookieDomain)) {
+      throw new Error('Cookie domain not declared in the manifest')
+    }
+    try {
+      log.info(
+        `getCookie on this domain : ${cookieDomain} for account ${this.startContext.account.id}`
+      )
+      const cookies = await CookieManager.get(cookieDomain)
+      log.info('CookieManager.get =>', cookies)
```

Comment:
It should be debug() or something like that. Since we'll start working pretty soon on this subject, we should not log critical personal data. Only on debug mode.